api/models/HHistory.js

```diff
@@ -0,0 +1,80 @@
+/**
+ * HHistory.js
+ *
+ * @description :: hHistory model
+ * @docs        :: http://sailsjs.org/#!documentation/models
+ */
+
+module.exports = {
+  tableName: 'h_history',
+
+  primaryKey: 'dateReviewed',
+
+  attributes: {
+    id: {
+      type: 'number',
+      autoIncrement: true,
+      columnName: 'id',
+      unique: true,
```

Comment:
nop, you can have multiple H_history with the same id. you can have multiple "snapshots" for the same history no ?

Reply:
We took HDocument and HDescription as a base for HHistory and they have `autoIncrement : true` and `unique : true` but we will change this,